ci/do_ci.sh

```diff
@@ -5,11 +5,13 @@ set -e
 export BUILDIFIER_BIN="/usr/local/bin/buildifier"
 
 function do_build () {
+    setup_gcc_toolchain
     bazel build $BAZEL_BUILD_OPTIONS --verbose_failures=true //:nighthawk_client //:nighthawk_test_server \
-        //:nighthawk_service
+    //:nighthawk_service
```

Comment:
Nit: I think the indent was clearer before

Reply:
Fixed. It would be great to have check_format/fix for bash :)